packages/cli/src/cli.js

```diff
@@ -2,4 +2,4 @@
 'use strict';
 
 /* eslint-disable */
-require('./lib/boot-cli.js');
+import('./lib/boot-cli.js');
```

Comment:
Should this be a typescript file already?

Reply:
I belive the original blocker was related to esbuild. not sure and not a priority